src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/EnvelopeValidator.java

```diff
@@ -86,21 +88,38 @@ public static void assertEnvelopeContainsOcrDataIfRequired(InputEnvelope envelop
      * @param pdfs     to assert against
      */
     public static void assertEnvelopeHasPdfs(InputEnvelope envelope, List<Pdf> pdfs) {
+        List<String> problems = new ArrayList<>();
+
+        List<String> duplicateFileNames =
+            envelope
+                .scannableItems
+                .stream()
+                .map(it -> it.fileName)
+                .collect(groupingBy(it -> it, counting()))
+                .entrySet()
+                .stream()
+                .filter(it -> it.getValue() > 1)
+                .map(it -> it.getKey())
+                .collect(toList());
+        
+        if (!duplicateFileNames.isEmpty()) {
+            problems.add("Duplicate scanned items file names: " + String.join(", ", duplicateFileNames));
```

Comment:
Perhaps clearer if notified where exactly? Could even get the `zipFileName` there if needed

```suggestion
            problems.add("Duplicate scanned items in metafile: " + String.join(", ", duplicateFileNames));
```

Reply:
Currently existing errors are: 
>Not declared PDFs: ...
and
>Missing PDFs:...

Reply:
Based on exception type and appropriate error code is set on the notification.

Reply:
Yea thought so. Still makes clear duplicate names **are** in metafile itself. Don't mind hence approved